src/main/java/com/mitchej123/hodgepodge/mixins/early/minecraft/MixinItemPotion_FixRomanNumerals.java

```diff
@@ -0,0 +1,39 @@
+package com.mitchej123.hodgepodge.mixins.early.minecraft;
+
+import com.mitchej123.hodgepodge.Common;
+import com.mitchej123.hodgepodge.util.RomanNumerals;
+import net.minecraft.item.ItemPotion;
+import net.minecraft.potion.PotionEffect;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+
+@Mixin(ItemPotion.class)
+public class MixinItemPotion_FixRomanNumerals {
```

Comment:
Unneccessary for the same reason as with enchants. Just add the different localizations to the lang file (with `potion.potency` instead of `enchantment.level`).

Reply:
Which is a finite amount of levels (128), so doing it via localizations would still be a viable way.

Reply:
It's better to write a method that computes the roman numbers instead of taking hours to write them manually and eventually fail

Reply:
1) It wouldn't take hours, at most 60 minutes
2) You shouldn't do it by hand, you should automate it (you mostly have the code already...)
3) You are doing it *once* and not millions of times for the same 128 numbers